packages/react-native-reanimated/Common/cpp/NativeModules/NativeReanimatedModule.cpp

```diff
@@ -736,7 +736,19 @@ void NativeReanimatedModule::performOperations() {
             }
 #endif
           }
-          return cloneShadowTreeWithNewProps(oldRootShadowNode, propsMap);
+
+          auto rootNode =
+              cloneShadowTreeWithNewProps(oldRootShadowNode, propsMap);
+
+          // Mark the commit as Reanimated commit so that we can distinguish it
+          // in ReanimatedCommitHook.
+
+          auto reaShadowNode =
```

Comment:
Let's avoid using "rea" if needed.
```suggestion
          auto markedShadowNode =
```
or
```suggestion
          auto shadowNodeWithTrait =
```

Reply:
Right, how about `ReanimatedTraitAccessEnabledShadowNode`? That's probably too long.